ProcessMaker/modeler
# Add BPMN diagram validation to status bar

Fixes #196.

**How to test:**
 * Click the "Validate Diagram" button in the top right of the grid.
 * With only the start event present, there should be two BPMN lint errors logged to the console.
 * If you add an end event and connect the start event to it, all validation errors should be resolved.


**Important changes:**
 * Added two new packages (make sure to run `npm i` before testing):
   - https://github.com/bpmn-io/bpmnlint
   - https://github.com/nikku/bpmnlint-loader
 * This PR modifies the `ModelerApp.vue` file, which is not exported after building. A separate PR will need to be created to merge the changes to `ModelerApp.vue` into https://github.com/ProcessMaker/bpm.
